pkg/api/serialization_test.go

```diff
@@ -174,6 +174,16 @@ var nonRoundTrippableTypes = sets.NewString(
 	"WatchEvent",
 )
 
+// verify all external group/versions have ListOptions registered.
+func TestListOptionsRegistered(t *testing.T) {
+	for _, group := range testapi.Groups {
+		gv := group.GroupVersion()
+		if _, err := api.Scheme.New(gv.WithKind("ListOptions")); err != nil {
```

Comment:
Expand to catch the other types, also watch events (I guess there are a few groups not covered by that, but I think they have their own schemes).